src/flow/src/lib.rs

```diff
@@ -25,7 +25,7 @@
 // #![cfg_attr(not(test), warn(clippy::indexing_slicing))]
 
 // allow unused for now because it should be use later
-mod adapter;
+pub mod adapter;
```

Comment:
If we only want `FlownodeOptions` to be public, we can simply write:

```
mod adapter;

pub use crate::adapter::FlownodeOptions;
```

This way, we don't need to export the entire `adapter` module.